docs/monorepo.md

```diff
@@ -1,11 +1,15 @@
-Publishing Packages with the Monorepo
-=====================================
+# Publishing with the Monorepo
 
 Calypso is a monorepo. In addition to the Calypso application, it also hosts a number of independent modules that are published to NPM.
 
 ## Module Layout
 
-These modules live under the `packages` directory, one folder per module.
+These modules live under the `packages` and `apps` directories, one folder per module.
+
+Two different directories for modules are:
+
+`/packages` — projects and libraries that we use in Calypso but also might publish as independent outputs or packages.
+`/apps` — projects that we _do not_ use directly in Calypso and publish as independent outputs or packages.
```

Comment:
The distinction I had in mind was more that stuff in packages represented packages we would publish to npm, and usage in Calypso didn't really matter (though if they're not used in Calypso, why are they here at all?) Stuff in "apps" was for the stuff that represented an "app", so something that produced something like a tarball that you could install by hand elsewhere. Though I guess there's no reason it couldn't also be published to npm...

Reply:
Thanks for elaborating, very helpful! Updated: https://github.com/Automattic/wp-calypso/pull/32137/commits/208ed7ddfe657f75dcd7f144f95e0f79f9097b1c